packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-actions.ts

```diff
@@ -34,6 +34,36 @@ function generatedAnswerActions(selector: GeneratedAnswerSelector) {
         .retryButton()
         .click()
         .logAction('When clicking on the retry button of the generated answer'),
+
+    clickFeedbackOption: (index: number) =>
+      selector
+        .feedbackOption(index)
+        .click({force: true})
+        .logAction(`When clicking the feedback option at the index ${index}`),
+
+    clickFeedbackSubmitButton: () =>
```

Comment:
```suggestion
    clickFeedbackSubmit: () =>
```
Do we need to call all of them XButton? IMO it's more readable without it